components/Navigation/Navigation.js

```diff
@@ -12,43 +12,21 @@ export const Navigation = () => {
     setIsMenuOpen(!isMenuOpen);
   };
 
-  const renderMobileMenu = () => (
-    <nav className="bg-accent-200 absolute top-[52px] md:top-[76px] lg:invisible right-0 p-3 z-50">
-      <ul className="flex flex-col">
-        <li className="divide-y-2 divide-black">
-          <Link href="/new-article">
-            <a className="divide-y-2 divide-black p-3">Nový článek</a>
-          </Link>
-        </li>
-        <li className="divide-y-2 divide-black">
-          <Link href="/profile">
-            <a className="divide-y-2 divide-black p-3">Profil</a>
-          </Link>
-        </li>
-        <li className="divide-y-2 divide-black">
-          <Link href="/logout">
-            <a className="divide-y-2 divide-black p-3">Odhlasit se</a>
-          </Link>
-        </li>
-      </ul>
-    </nav>
-  );
-
   const hamburgerTopClass = classNames({ 'rotate-45': isMenuOpen });
   const hamburgerMiddleClass = classNames({ 'opacity-0': isMenuOpen });
   const hamburgerBottomClass = classNames({ '-rotate-45': isMenuOpen });
+  const hiddenMobileMenu = classNames({ hidden: !isMenuOpen });
```

Comment:
You can also do here 
```suggestion
  const navClassName = classNames({ hidden: !isMenuOpen }, 'bg-accent-200 absolute top-[52px] md:top-[76px] right-0 z-50 lg:bg-transparent lg:relative lg:flex-1 lg:flex lg:top-0');
```